veneer/percona-server-mongodb-operator.yaml

```diff
@@ -4,7 +4,9 @@ GenerateMinorChannels: false
 Fast:
   Bundles:
   - Image: quay.io/operatorhubio/percona-server-mongodb-operator:v1.15.0
+  - Image: quay.io/operatorhubio/percona-server-mongodb-operator:v1.16.0
 Stable:
   Bundles:
   - Image: quay.io/operatorhubio/percona-server-mongodb-operator:v1.14.0
```

Comment:
We should also remove 1.14.0, since we want to support only the 2 most recent versions

Reply:
Done